### ISSUE TITLE
Broken syntax highlighting

### ISSUE THREAD

**Issue by anuragsoni** [realworldocaml/book#2960]
Hi,

The syntax highlighting seems to be broken on some chapters. So far i've seen it on the chapter about menhir and the chapter about handling json data.

**Chapter about handling json data**

<img width="821" alt="screen shot 2018-05-27 at 6 57 36 pm" src="https://user-images.githubusercontent.com/5814535/40591470-f81a6fe8-61df-11e8-8ae4-1047deb5c5e3.png">

**Chapter about Menhir**

<img width="811" alt="screen shot 2018-05-27 at 6 55 37 pm" src="https://user-images.githubusercontent.com/5814535/40591471-f8291322-61df-11e8-9ee3-42705f5e6205.png">

**Comment by yminsky** [comment]
That is deeply odd. @samoht any idea why the syntax highlighting would be cutting in and out like that?

**Comment by anuragsoni** [comment]
If it helps the browser was the latest stable Google Chrome.

**Comment by ulugbekna** [comment]
I think this problem persists as of today. I initially thought this was a problem on my end but got a confirmation that another person had it too.

![image](https://user-images.githubusercontent.com/16353531/53684007-c59f0f80-3d29-11e9-9d51-9dcbadd8ae02.png)

**Comment by yminsky** [comment]
Agreed. I can confirm that the json chapter is broken as per the original issue. @samoht, can you take a look?
